src/Windows.Core/DEVMODE.cs

```diff
@@ -0,0 +1,1326 @@
+        /// This value is also known as the display device's vertical refresh rate. Display drivers use this member.
+        /// It is used, for example, in the ChangeDisplaySettings function. Printer drivers do not use this member.
+        /// </summary>
+        /// <remarks>
+        /// When you call the <c>User32.EnumDisplaySettings</c> function, the <see cref="dmDisplayFrequency"/> member may return with the value 0 or 1.
+        /// These values represent the display hardware's default refresh rate.
+        /// This default rate is typically set by switches on a display card or computer motherboard, or by a configuration program that does not use display functions such as <c>User32.EnumDisplaySettings</c>.
+        /// </remarks>
+        /// <remarks>
+        /// This member is not used for printers.
+        /// </remarks>
+        [FieldOffset(148)]
+        public uint dmDisplayFrequency;
+
+        /// <summary>
+        /// Specifies one of the <see cref="DEVMODE_DeviceIcmMethodOptions"/> constants.
+        /// </summary>
+        [FieldOffset(188)]
+        public DEVMODE_DeviceIcmMethodOptions dmICMMethod;
+
+        /// <summary>
+        /// Specifies one of the <see cref="DEVMODE_DeviceIcmIntentOptions"/> constants.
+        /// </summary>
+        [FieldOffset(192)]
+        public DEVMODE_DeviceIcmIntentOptions dmICMIntent;
+
+        /// <summary>
+        /// Specifies one of the <see cref="DEVMODE_DeviceMediaTypeOptions"/> constants.
+        /// </summary>
+        [FieldOffset(196)]
+        public DEVMODE_DeviceMediaTypeOptions dmMediaType;
+
+        /// <summary>
+        /// Specifies one of the <see cref="DEVMODE_DitherTypeOptions"/> constants.
+        /// </summary>
+        [FieldOffset(200)]
+        public DEVMODE_DitherTypeOptions dmDitherType;
+
+        /// <summary>
+        /// Not used; must be zero.
+        /// </summary>
+        [FieldOffset(204)]
+        public uint dmReserved1;
+
+        /// <summary>
+        /// Not used; must be zero.
+        /// </summary>
+        [FieldOffset(208)]
+        public uint dmReserved2;
+
+        /// <summary>
+        /// This member must be zero.
+        /// </summary>
+        [FieldOffset(212)]
+        public uint dmPanningWidth;
+
+        /// <summary>
+        /// This member must be zero.
+        /// </summary>
+        [FieldOffset(216)]
+        public uint dmPanningHeight;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DEVMODE"/> struct
+        /// with the <see cref="dmSize" /> field initialized.
+        /// </summary>
+        /// <returns>
+        /// An instance of the structure.
+        /// </returns>
+        public static DEVMODE Create() => new DEVMODE { dmSize = (ushort)sizeof(DEVMODE) };
+    }
+
+    [Flags]
+    [SuppressMessage("Compiler", "SA1201:Elements should appear in the correct order", Justification = "Special")]
```

Comment:
This enum has as SuppressMessage but others don't....

Reply:
It's only complaining about the first one.